### ISSUE TITLE
Google Sheets read values as middle step

### ISSUE THREAD

**Issue by christophd** [syndesisio/syndesis#5132]
## This is a...

<pre><code>
[x] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description

Google Sheets obtain values operation is only provided as start connection action. We should also provide this as a middle step action in order to be able to read values from Google Sheets in the middle of an integration.

**Comment by christophd** [comment]
@TovaCohen something we need to add to the docs, too. Sorry, for the late notice. 

It is the same operation as the get values from spreadsheet operation that can be used as a start connection in an integration. There are some slight differences in configuration properties though. The retrieve values from spreadsheet operation as middle step is missing the properties `delay`, `splitResults` and `maxResults`.

**Comment by TovaCohen** [comment]
No worries. Thanks Christoph. I updated the doc here:
https://docs.google.com/document/d/1KeBgAP6UU-fTgMp3zysQKIwNW-GmmCADlX5yrOGulw4/edit?usp=sharing

And also tagged you in a comment in that document. 
I chose to edit the content for the topic that describes how to trigger an integration by obtaining values from a spreadsheet. The other option is to create a new topic. That is, there would be one topic for obtaining values in a start connection, and another topic for obtaining values in a middle connection. Let me know if you think two topics is better. 

I created this Jira to track the work:
https://issues.jboss.org/browse/FUSEDOC-3236
